.circleci/config.yml

```diff
@@ -107,13 +107,36 @@ jobs:
           paths:
             - ~/.gradle
           key: gradle-{{ checksum "build.gradle" }}
+# Integration tests
+  integration-tests:
+    environment:
+      # Configure the JVM and Gradle to avoid OOM errors
+      _JAVA_OPTIONS: "-Xmx1g"
+      GRADLE_OPTS: "-Dorg.gradle.daemon=false -Dorg.gradle.workers.max=2"
+    docker:
+      - image: openkbs/jdk-mvn-py3
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - gradle-w3c-{{ checksum "build.gradle" }}
+      - run:
+          name: Integration Tests
+          command: cd integration_tests/tracecontext && ./tracecontext-integration-test.sh
+      - run:
+          <<: *publish_snapshots_task
+      - save_cache:
+          paths:
+            - ~/.gradle
+          key: gradle-w3c-{{ checksum "build.gradle" }}
 
 workflows:
   version: 2
   build_and_test:
     jobs:
       - build
       - java11
+      - integration-tests
```

Comment:
If we have failing tests, won't this fail every build until we get them fixed?

Reply:
get docker image with python3 and pip3 made my day crazy... still not be able to do that

Reply:
we can hold on that for a follow-up if you want. I'd be happy getting the tests in as a first step, then automating the running of them as a second step.

Reply:
I removed automation